src/plugins/translate/plugin.translate.js

```diff
@@ -372,42 +375,54 @@ export class BrTranslatePanel extends LitElement {
-            ${this.fromLanguages.map((lang) => {
-      return html`<option
-                value="${lang.code}"
-                ?selected=${lang.code == this.detectedFromLang}
-              >${lang.name}</option>`;
-    },
-    )}
+          <span style="font-size: 12px;color: #ccc;">Translate To</span>
+          <select id="lang-to" name="to" class="lang-select" style="display:block; width:100%;" @change="${this._onLangToChange}">
+          ${sortBy(this.toLanguages, ((lang) => lang.name.toLowerCase()
+    )).map((lang) => {
+      return html`<option value="${lang.code}" 
+                        ?selected=${lang.code == this.detectedToLang}
+                        > ${lang.name ? lang.name : lang.code} </option>`;
+    })
+          }
           </select>
         </label>
       </div>
-      <div class="panel panel--to">
-        <label>
-          To
-          <select id="lang-to" name="to" class="lang-select" @change="${this._onLangToChange}">
-            ${this.toLanguages.map(
-      lang => html`<option value="${lang.code}">${lang.name}</option>`,
-    )}
-          </select>
-        </label>
-      </div>
-      <div class="panel panel--start">
-        <label>
-          Toggle Translation
+
+      <div class="panel panel--start" style="text-align: right;padding: 0 10px;/*! font-size: 18px; */margin-top: 10px;">
             <button class="start-translation-brn" @click="${this._toggleTranslation}">
-              ${this.userTranslationActive ? "Disable" : "Enable"}
+              ${this.userTranslationActive ? "Stop Translating" : "Translate"}
             </button>
-        </label>
       </div>
-      <div class="footer" id="status"></div>
-      <br/>
-      <div class="disclaimer" id="disclaimerMessage"> ${this.disclaimerMessage} </div>
+
+      <div class="panel panel--from" style="font-size: 12px;color: #ccc;text-align: center;padding: 8px 10px;">
+        <details style="display: contents">
+          <summary style="text-decoration: underline white; cursor:pointer; display:inline-block"> 
+            <i>
+              Source: ${this._getLangName(this.detectedFromLang)} ${this.prevSelectedLang ? "" : "(detected)"} 
+            </i> Change 
+          </summary>
+          <select id="lang-from" name="from" class="lang-select" value=${this.detectedFromLang} @change="${this._onLangFromChange}" style="width:65%; margin-top: 3%; margin-bottom: 3%">
+          ${sortBy(this.fromLanguages, ((lang) => lang.name.toLowerCase()
+          )).map((lang) => {
+            return html`<option value="${lang.code}"
+                      ?selected=${lang.code == this.detectedFromLang}
+                      >${lang.name ? lang.name : lang.code} </option`;
+          })
+          }
+          </select>
+      </details>
+      <div class="footer" id="status" style="margin-top:5%">
+      ${this._statusWarning()}
+      </div>
```

Comment:
I don't think we need to show this warning. It's a bit daunting to be presented with a warning as soon as you open the panel! Since English will likely be the default. Instead, let's when "Translate" is clicked, show a `window.alert` with this error message.